{
    "name": "​DataSource​Exception",
    "namespace": "DataSource",
    "accessModifier": "global",
    "definitionModifier": "",
    "withSharing": false,
    "inheritedSharing": false,
    "isEnum": false,
    "enumValues": [],
    "extendsType": "Exception",
    "isInterface": false,
    "implements": [],
    "classes": {},
    "enums": {},
    "fields": [],
    "constructors": [],
    "methods": [],
    "description": "Throw this exception to indicate that an error occurred while communicating with an external data source.",
    "docLink": "https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_DataSource_exceptions.htm"
}